### DISCUSSION TITLE
GSTools v1.3 with meshio support

### DISCUSSION THREAD

**Discussion by MuellerSeb** [nschloe/meshio#1071] [Show and tell]
:tada: I'd like to announce GSTools v1.3, the geo-statistical toolbox for Python, which brings an abundancy of new features: Swiss-Army-Knife for Kriging, conditioned fields, geographical coordinates support, directional variogram estimation & auto-binning, arbitrary dimensions, data normalizers & trends, Interfaces to meshio, PyVista & PyKrige and much much more. Check out the Release Notes and the rich Documentation. Available on PyPI and Conda. Give it a shot!
https://github.com/GeoStat-Framework/GSTools

Here a [meshio example](https://geostat-framework.readthedocs.io/projects/gstools/en/stable/examples/01_random_field/05_mesh_ensemble.html#sphx-glr-examples-01-random-field-05-mesh-ensemble-py) with an ensemble of random fields:

![sphx_glr_05_mesh_ensemble_002](https://user-images.githubusercontent.com/19690642/114847664-3a384980-9dde-11eb-90c7-c8f4517d7448.png)

**Comment by nschloe** [comment]
Super cool!
